lookout/style/cloner.py

```diff
@@ -0,0 +1,83 @@
+"""Cloning utilities for git repositories."""
+
+import logging
+import os
+from typing import Dict, Optional, Sequence, Tuple
+
+from dulwich import porcelain
+from joblib import delayed, Parallel
+
+
+class ParallelWriteToLogs(Parallel):
+    """Parallel which logs progress using logger."""
+
+    _log = logging.getLogger("Parallel")
+
+    def _print(self, msg, msg_args):
+        """Display the message using `self._log.info`."""
+        # XXX: Not using the logger framework: need to
+        # learn to use logger better.
+        if not self.verbose:
+            return
+        self._log.info(msg, *msg_args)
```

Comment:
```suggestion
        self._log.log(self.verbose, msg, *msg_args)
```
and remove all the sh*t from the above :D